packages/iov-bns/src/encode.ts

```diff
@@ -142,6 +149,16 @@ function buildAddAddressToUsernameTx(tx: AddAddressToUsernameTx): codecImpl.app.
   };
 }
 
+function buildCreateMultisignatureTx(tx: CreateMultisignatureTx): codecImpl.app.ITx {
+  return {
+    createContractMsg: {
+      participants: tx.participants.map(participant => participant as codecImpl.multisig.IParticipant),
```

Comment:
hmm, why this map? Can't `Participant` be assigned to `codecImpl.multisig.IParticipant` by default?